docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,62 @@
+- [PMM-10270](https://jira.percona.com/browse/PMM-10270) -[Backup Management]: Enhanced MongoDB Backup Management to support restoring to different replica set clusters with the same configuration.
+- [PMM-11999](https://jira.percona.com/browse/PMM-11999), [PMM-12257](https://jira.percona.com/browse/PMM-12257) - The [Advisors] page now contains a new Technology column that shows the database type for each advisor. This helps you distinguish between MongoDB, MySQL, and PostgreSQL advisors.
+- [PMM-11637](https://jira.percona.com/browse/PMM-11637) - In PMM 2.39.0, we updated the interface to reflect the new PMM logo as per the new branding.
+- [PMM-11206](https://jira.percona.com/browse/PMM-11206) - [Alerting]: Enhanced the **Add rule template** dialog box to support YAML content for multiple alert templates. Alert templates added in bulk are displayed individually on the **Alert rule templates** page.
+- [PMM-12070](https://jira.percona.com/browse/PMM-12070) - We have updated the deprecated [Boomtable plugin](https://github.com/percona/grafana-dashboards/tree/main/panels/yesoreyeram-boomtable-panel) to the Grafana table plugin.
+
+ 
+## Bugs Fixed
+
+- [PMM-12151](https://jira.percona.com/browse/PMM-12151) - [QAN]: The Explain plan was not displayed in a few scenarios. The issue has been resolved now.
+- [PMM-11658](https://jira.percona.com/browse/PMM-11658) - pmm-agent was crashing on AMI and OVF distributions.
+- [PMM-11992](https://jira.percona.com/browse/PMM-11992) - Users who configured their Postgres databases with names other than **Postgres** could not add them. The issue has been resolved now.
+- [PMM-12231](https://jira.percona.com/browse/PMM-12231) - PMM prevented the users from installing, updating, or removing plugins. The issue has been resolved now.
+- [PMM-12263](https://jira.percona.com/browse/PMM-12263) - On the MongoDB Cluster Summary dashboard, the **Replication Lag by Set** graph showed the wrong value if **rs** was in the primary-secondary-arbiter (PSA) configuration. It calculated the arbiter's lag from the beginning of the UNIX time (currently >54 years instead of a few seconds). The issue has been resolved now.
+- [PMM-12280](https://jira.percona.com/browse/PMM-12280) - Fixed an issue where the **Service Summary Panel** on Home Dashboard used the incorrect units to display `Available Memory`.
+- [PMM-10665](https://jira.percona.com/browse/PMM-10665) - Fixed issue caused QAN to show incorrect values when setting non-default values for performance schema. Make sure to restart pmm-agent after changing the settings for perfschema in MySQL.
+- [PMM-10391](https://jira.percona.com/browse/PMM-10391) - PMM did not work with external clickhouse versions greater than 22.5.2. The issue has been resolved now.
+
+## Known issues
+
+**PMM does not allow to install, update or remove plugins**
```

Comment:
@atymchuk , @rasika-chivate  let's keep this section only if users on 2.39 are still affected. 
from the Jira I don't see this information.

Reply:
I have removed this from Known issues.